frontend/src/main/scala/akka/viz/frontend/components/components.scala

```diff
@@ -325,7 +332,7 @@ class MessagesPanel(selectedActors: Var[Set[String]]) extends Component with Pre
         table(
           cls := "table table-striped table-hover",
           thead(
-            tr(th("From"), th("To"), th("Class"))
+            tr(th(), th("From"), th("To"), th("Class"))
```

Comment:
Maybe put it somewhere inline, next to message class? That table is already too wide ;)